src/ui/preview/PreviewContainer.vue

```diff
@@ -108,6 +108,9 @@ export default {
       this.addExistingViewBackToParent();
     }
   },
+  updated() {
+    this.getActionsCollection(this.view);
+  },
```

Comment:
This is an unconventional change and might raise some eyebrows in the future. We should add a `// FIXME:` and a brief note explaining why it's here. We should also reference the issue URL

Reply:
You know what? This totally escaped my mind. We might be able to just use `await nextTick()` in the `mounted()` hook before running `this.getActionsCollection(this.view);`. That way we're not running that on every update, even though I think it technically only gets run once anyway.

Reply:
Unfortunately, nextTick didn't bail us out this time so I left the updated lifeycle hook and added a fixme to give more context.

Reply:
Damn! Ok, sounds good